app-core.js

```diff
@@ -23,10 +23,18 @@ function setupExpressServer() {
   app.use(express.json());
 
   const viewsPath = getViewsPath();
+
+  if (process.env.NODE_ENV === "production") {
+    const { getOctoFarmUiPath } = require("@octofarm/client");
+    const bundlePath = getOctoFarmUiPath();
+    app.use("/assets/dist", express.static(bundlePath));
```

Comment:
So we have to expand this logic later. We could offer a direct bundle download without NPM as backup, or use a github release asset. Whatever we feel like, but it needs more user interactivity.

I noted this in my mind and expect to enhance it later! Oh btw remember the `NODE_ENV` also needs to be in production by default, made sure to be correct... whatever is most robust.